pkg/core/native/native_test/neo_test.go

```diff
@@ -331,6 +333,19 @@ func TestNEO_RecursiveGASMint(t *testing.T) {
 	neoValidatorInvoker.Invoke(t, true, "transfer", e.Validator.ScriptHash(), c.Hash, int64(1), nil)
 }
 
+func TestNEO_GetCommitteeAddress(t *testing.T) {
+	neoValidatorInvoker := newNeoValidatorsClient(t)
+	e := neoValidatorInvoker.Executor
+	standByCommitteePublicKeys, err := keys.NewPublicKeysFromStrings(e.Chain.GetConfig().StandbyCommittee)
+	require.NoError(t, err)
+	sort.Sort(standByCommitteePublicKeys)
+	expectedCommitteeAddress, err := smartcontract.CreateMultiSigRedeemScript(len(standByCommitteePublicKeys)/2+1, standByCommitteePublicKeys)
+	require.NoError(t, err)
+	stack, err := neoValidatorInvoker.TestInvoke(t, "getCommitteeAddress")
+	require.NoError(t, err)
+	require.Equal(t, hash.Hash160(expectedCommitteeAddress).BytesBE(), stack.Pop().Item().Value().([]byte))
```

Comment:
expectedCommitteeAddress is []byte (because of CreateMajorityMultiSigRedeemScript(publicKeys keys.PublicKeys) ([]byte, error)) so it doesn't have BytesBE()

Reply:
Sure, my bad, we should hash it.